src/_gettsim/taxes/lohnst.py

```diff
@@ -154,50 +288,17 @@ def vorsorgepauschale_y_ab_2010(  # noqa: PLR0913
     )
 
     # 2. Krankenversicherungsbeiträge, §39b (2) Nr. 3b EStG.
-    # For health care deductions, there are two ways to calculate.
+    # For health care deductions, there are two ways to calculate
+    # the deuctions.
     # a) at least 12% of earnings of earnings can be deducted,
     #    but only up to a certain threshold
+    #  b) Take the actual contributions (usually the better option),
+    #   but apply the reduced rate
 
-    bruttolohn_kv = min(
-        12 * bruttolohn_m,
-        12 * sozialv_beitr_params["beitr_bemess_grenze_m"]["ges_krankenv"]["ost"],
-    )
-    vorsorg_kv_option_a_basis = (
-        eink_st_abzuege_params["vorsorgepauschale_mindestanteil"] * bruttolohn_kv
-    )
-
-    if steuerklasse == 3:
-        vorsorg_kv_option_a_max = eink_st_abzuege_params["vorsorgepauschale_kv_max"][
-            "steuerklasse_3"
-        ]
-    else:
-        vorsorg_kv_option_a_max = eink_st_abzuege_params["vorsorgepauschale_kv_max"][
-            "steuerklasse_nicht3"
-        ]
-
-    vorsorg_kv_option_a = min(vorsorg_kv_option_a_max, vorsorg_kv_option_a_basis)
-
-    # b) Take the actual contributions (usually the better option),
-    #   but apply the reduced rate!
-
-    if ges_pflegev_zusatz_kinderlos:
-        beitr_satz_pflegev = (
-            sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["standard"]
-            + sozialv_beitr_params["beitr_satz"]["ges_pflegev"]["zusatz_kinderlos"]
-        )
-    else:
-        beitr_satz_pflegev = sozialv_beitr_params["beitr_satz"]["ges_pflegev"][
-            "standard"
-        ]
-
-    vorsorg_kv_option_b = bruttolohn_kv * (
-        sozialv_beitr_params["beitr_satz"]["ges_krankenv"]["ermäßigt"] / 2
-        + ges_krankenv_zusatzbeitr_satz / 2 / 100
-        + beitr_satz_pflegev
-    )
+    vorsorg_kv = max(vorsorg_kv_option_a, vorsorg_kv_option_b)
 
     # add both RV and KV deductions. For KV, take the larger amount.
-    out = vorsorg_rv + max(vorsorg_kv_option_a, vorsorg_kv_option_b)
+    out = vorsorg_rv + vorsorg_kv
```

Comment:
```suggestion
    out = vorsorg_rentenv + vorsorg_krankenv
```
(please change everywhere, we must not introduce different abbreviations of the same thing)